app/assets/stylesheets/pages/_home.css.scss

```diff
@@ -72,11 +72,9 @@
     img {
       border-radius: 4px;
       box-shadow: 0 0 1px $gray;
-      margin-top: 1rem;
+      margin: 1rem 0 2rem 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `1rem 0 2rem` instead of `1rem 0 2rem 0`